getting-started/exchange/building-blocks.md

```diff
@@ -0,0 +1,169 @@
+# Building blocks
+
+Building blocks are a collection of files, that can be used to supplement a process or other parts of an implementation.
+These can consist of forms, form flows, plugins, custom front-end, and back-end code. Unlike [process blueprints](blueprints.md),
+building blocks do not contain a process definition. For example, a plugin that allows sending messages via Slack.
```

Comment:
The example is unclear because of the sentence excluding process definitions. The example (plugin) might be part of the exclusion.